src/ucs/arch/x86_64/global_opts.c

```diff
@@ -17,28 +18,39 @@ ucs_config_field_t ucs_arch_global_opts_table[] = {
 #if ENABLE_BUILTIN_MEMCPY
   {"BUILTIN_MEMCPY_MIN", "auto",
    "Minimal threshold of buffer length for using built-in memcpy.",
-   ucs_offsetof(ucs_arch_global_opts_t, builtin_memcpy_min), UCS_CONFIG_TYPE_MEMUNITS},
+   ucs_offsetof(ucs_arch_global_opts_t, builtin_memcpy_min),
+   UCS_CONFIG_TYPE_MEMUNITS},
 
   {"BUILTIN_MEMCPY_MAX", "auto",
    "Maximal threshold of buffer length for using built-in memcpy.",
-   ucs_offsetof(ucs_arch_global_opts_t, builtin_memcpy_max), UCS_CONFIG_TYPE_MEMUNITS},
+   ucs_offsetof(ucs_arch_global_opts_t, builtin_memcpy_max),
+   UCS_CONFIG_TYPE_MEMUNITS},
 #endif
+  {"NT_BUFFER_TRANSFER_MIN", "auto",
+   "Minimal threshold of buffer length for using nt-buffer-transfer.",
```

Comment:
maybe in help replace `nt-buffer-transfer` by `non-temporal buffer`

Reply:
Will do.